Source/Mosa.Tool.Launcher/MainWindow.axaml.cs

```diff
@@ -114,7 +116,7 @@ public void Initialize(string[] args)
 		PlatformRegistry.Add(new Platform.ARMv8A32.Architecture());
 
 		// Initialize settings
-		UpdateSettings();
+		UpdatePaths();
```

Comment:
Is this change correct? If so, update the comment

Reply:
The change is correct, I'll update the comment.